### ISSUE TITLE
ask ai

### ISSUE THREAD

**Issue by TriggerMinds** [TriggerMinds/kissmylilstar#124]
[What's the best prompt to analyze images

What would be your best prompt to analyze what is in an image.

For example, an installation of a cable, what's the best prompt to analyze and see if the cable is properly ran. Or if screws are all the way in properly...]

that was the question posed by a reddit user. my thought is this..what is important to the AI? what does it thrive on?

role playing
examples
requirements
constraints
clear instructions 

bottom line...context is Everything. with this in mind what do you recommend. provide three optimal prompts best suited to the challenge presented by the user, each in markdown format and displayed in code blocks for easy copying..

**Comment by TriggerMinds** [comment]
blend of all three into one
